docs/data-sources/machine_types.md

```diff
@@ -10,7 +10,11 @@ description: |-
 
 List of machine types
 
+## Example Usage
 
+```terraform
```

Comment:
Those docs are autogenerated 
in order to add an example for a data source/ resource you should add a directory in `generate_example_usages` like  [rosa_operator_roles data source](https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/generate_example_usages/data-sources/rhcs_rosa_operator_roles/data-source.tf)  and [ cluster_rosa_classic resource](https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/generate_example_usages/resources/rhcs_cluster_rosa_classic/resource.tf)
Then run `make generate`

Reply:
@bardielle thank you for reviewing. example updated in `generate_example_usage`